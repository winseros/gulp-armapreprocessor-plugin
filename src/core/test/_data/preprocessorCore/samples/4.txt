#define M1 abcd

#ifdef M1
AAA
#endif

M1

#define M2 aa\
           bb\
           cc

M2

#define M3(P1) P1
#define M4(P1, AA , B ,C) P1##AA##B##C