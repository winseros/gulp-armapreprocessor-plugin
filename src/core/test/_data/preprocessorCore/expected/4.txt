


AAA


abcd





aa
           bb
           cc




v1;

xqvf



"A"